src/voting-contracts/migrations/6_proposal_migration.js

```diff
@@ -0,0 +1,5 @@
+const Proposal = artifacts.require("Proposal");
```

Comment:
`2_proposal_factory_migration.js` already deploys `proposal` instance

Reply:
Thanks for pointing out.